plugins/system76-launch/fu-system76-launch-device.c

```diff
@@ -138,11 +138,22 @@ fu_system76_launch_device_detach(FuDevice *device, FuProgress *progress, GError
 
 	/* generate a message if not already set */
 	if (fu_device_get_update_message(device) == NULL) {
+		GUsbDevice *usb_device = fu_usb_device_get_dev(FU_USB_DEVICE(device));
 		g_autofree gchar *msg = NULL;
+		const gchar *unlock_keys;
+		switch (g_usb_device_get_pid(usb_device)) {
+		case 0x0001: /* launch_1 */
+			unlock_keys = "Fn+Esc";
+			break;
+		default:
+			unlock_keys = "Left Ctrl+Right Ctrl+Esc";
```

Comment:
All future Launch keyboards will use this combo, that is why this is the default.